etc/kayobe/kolla/globals.yml

```diff
@@ -31,6 +31,11 @@ prometheus_node_exporter_extra_volumes:
   - "textfile:/var/lib/node_exporter/textfile_collector"
 prometheus_node_exporter_cmdline_extras: "--collector.textfile.directory=/var/lib/node_exporter/textfile_collector"
 
+# Set Ceph manager exporter endpoints
+prometheus_ceph_mgr_exporter_endpoints:
```

Comment:
If the mgrs group is empty this will end up as:

```
prometheus_ceph_mgr_exporter_endpoints:
```

Which is equivalent to setting it to a Python `None`. Kolla Ansible will then fail to template prometheus.yml:

```
{% for exporter in prometheus_ceph_mgr_exporter_endpoints %}
```